versions/3.0.3.md

```diff
@@ -3337,6 +3337,8 @@ flows:
 Lists the required security schemes to execute this operation.
 The name used for each property MUST correspond to a security scheme declared in the [Security Schemes](#componentsSecuritySchemes) under the [Components Object](#componentsObject).
 
+An empty Security Requirement Object with no properties (`{}`) is also valid. It can be used as part of a `security` array in an <a href="#openapi-object">OpenAPI Object</a> or an <a href="#operation-object">Operation Object</a>, along with other Security Requirement Objects, meaning that security with them is optional.
```

Comment:
This text is redundant as it is obvious that an empty object is valid (no required fields), and this PR explains what it means in the different locations it can be used. It should be either or. Any further explanation belongs outside in a separate tutorial.